### PR TITLE
432 update floodagent for stack

### PR DESCRIPTION
**Pull request by kokfoongcmcl**
Done
- Updated to use the latest base lib for latest time series namespace
- Blazegraph namespace specified with parameter in docker-compose
- All connections to RDB are closed properly with try-with-resource

Not done
- Update icon names for different water range/trends

### CODE REVIEW

File: Agents/FloodAgent/src/test/java/uk/ac/cam/cares/jps/agent/flood/TestUpdateStations.java [hunk 1]
```diff
@@ -128,8 +128,8 @@ public void testUploadDataToRDB() throws UnsupportedCharsetException, IOExceptio
 		api = mock(APIConnector.class);
 		when(api.getData(httpClient)).thenReturn(entity);
 		
-		ProcessedData processed_data = UpdateStations.processAPIResponse(api);
-		UpdateStations.uploadDataToRDB(tsClient, sparqlClient, processed_data);
+		// ProcessedData processed_data = UpdateStations.processAPIResponse(api);
+		// UpdateStations.uploadDataToRDB(tsClient, sparqlClient, processed_data);
```
Comment (markushofmeister) [inline Agents/FloodAgent/src/test/java/uk/ac/cam/cares/jps/agent/flood/TestUpdateStations.java:132]: Why did you comment out those 2 lines?

**Review by markushofmeister — COMMENTED** [review]
Looks fine to me, although I did not try running the agent (as I am still facing issues using the Feature Info Agent). Have you successfully done that?

**Comment by kokfoongcmcl** [comment]
> And am I supposed to update issue #432 after approving this PR or how is the workflow for that?

GitHub closes the issue automatically once this PR closes

**Comment by markushofmeister** [comment]
> > And am I supposed to update issue #432 after approving this PR or how is the workflow for that?
> 
> GitHub closes the issue automatically once this PR closes

However, we only want to close parts of the issue

**Comment by kokfoongcmcl** [comment]
> 

If github closes it we can reopen it

**Review by markushofmeister — APPROVED** [review]
Addressed changes look good. Implementation of icon-text in Postgis to be tracked as issues for later implementation

**Pull request merged**